apps/codeforafrica/src/payload/globals/Publication/PostTab.js

```diff
@@ -0,0 +1,95 @@
+const PostTab = {
+  label: "Post",
+  fields: [
+    {
+      type: "collapsible",
+      label: "Story",
+      fields: [
+        {
+          name: "stories",
+          label: "Recent stories",
+          type: "group",
+          localized: true,
+          fields: [
+            {
+              name: "showRecent",
+              label: "Show recent stories",
+              type: "checkbox",
+              required: true,
+              defaultValue: false,
+            },
+            {
+              name: "title",
+              label: "Title",
+              type: "text",
+              required: true,
+              defaultValue: "Recent Stories",
+              admin: {
+                condition: (_, data) => data.showRecent,
+              },
+            },
+            {
+              name: "primaryTag",
+              label: "Primary Tag",
+              type: "text",
+              required: true,
+              defaultValue: "stories",
+              admin: {
+                hidden: true,
+              },
+            },
```

Comment:
Thinking out loud, do we need this field if the group is already named "stories" or "opportunities" i.e. `group.name` matches `primaryTag`.

Reply:
I agree. We should remove this as it is not used anywhere.